src/services/ISessionStorage.ts

```diff
@@ -75,7 +75,7 @@ export interface ISessionStorage {
    * @param user The AppUser value used to get the related fiscalCode
    */
```

Comment:
We should update the doc as well

Reply:
done https://github.com/pagopa/io-backend/pull/995/commits/1216de4e971ce08cf7c7b7b5f9939dfdc4845153